op-node/p2p/rpc_api.go

```diff
@@ -1,63 +1,14 @@
 package p2p
 
 import (
-	"context"
-	"net"
-	"time"
-
-	"github.com/ethereum/go-ethereum/p2p/enode"
-	"github.com/libp2p/go-libp2p/core/network"
-	"github.com/libp2p/go-libp2p/core/peer"
-
-	"github.com/ethereum-optimism/optimism/op-node/p2p/store"
+	"github.com/ethereum-optimism/optimism/op-service/apis"
+	"github.com/ethereum-optimism/optimism/op-service/sources"
 )
 
-type PeerInfo struct {
-	PeerID          peer.ID  `json:"peerID"`
-	NodeID          enode.ID `json:"nodeID"`
-	UserAgent       string   `json:"userAgent"`
-	ProtocolVersion string   `json:"protocolVersion"`
-	ENR             string   `json:"ENR"`       // might not always be known, e.g. if the peer connected us instead of us discovering them
-	Addresses       []string `json:"addresses"` // multi-addresses. may be mix of LAN / docker / external IPs. All of them are communicated.
-	Protocols       []string `json:"protocols"` // negotiated protocols list
-	// GossipScore float64
-	// PeerScore float64
-	Connectedness network.Connectedness `json:"connectedness"` // "NotConnected", "Connected", "CanConnect" (gracefully disconnected), or "CannotConnect" (tried but failed)
-	Direction     network.Direction     `json:"direction"`     // "Unknown", "Inbound" (if the peer contacted us), "Outbound" (if we connected to them)
-	Protected     bool                  `json:"protected"`     // Protected peers do not get
-	ChainID       uint64                `json:"chainID"`       // some peers might try to connect, but we figure out they are on a different chain later. This may be 0 if the peer is not an optimism node at all.
-	Latency       time.Duration         `json:"latency"`
-
-	GossipBlocks bool `json:"gossipBlocks"` // if the peer is in our gossip topic
-
-	PeerScores store.PeerScores `json:"scores"`
-}
-
-type PeerDump struct {
-	TotalConnected uint                 `json:"totalConnected"`
-	Peers          map[string]*PeerInfo `json:"peers"`
-	BannedPeers    []peer.ID            `json:"bannedPeers"`
-	BannedIPS      []net.IP             `json:"bannedIPS"`
-	BannedSubnets  []*net.IPNet         `json:"bannedSubnets"`
-}
+// backwards compatibility
+var NamespaceRPC = sources.NamespaceRPC
```

Comment:
```suggestion
// backwards compatibility
var NamespaceRPC = sources.P2PNamespaceRPC
```
We should change it so that it's not ambiguous with other RPC namespaces that may be declared in the `sources` package.

Reply:
Fixed at https://github.com/ethereum-optimism/optimism/pull/15462/commits/fc1a78a224afd620a424fd0f15b41b8a8e030181